lib/linters.js

```diff
@@ -191,16 +193,28 @@ function badBindingExpressions(analyzer, is) {
       .concat(attributeBadBindingExpressions(analyzer, is));
 }
 
-function isProblematicAttribute(attr) {
-  var isLabel = p.hasTagName("for");
-  switch(attr.name.toLowerCase()) {
+function isNativeAttribute(name) {
+  switch (name) {
     case "class":
     case "style":
+    case "tabindex":
+    case "title":
+    case "style":
     case "href":
```

Comment:
From https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes we should add:

accesskey
contenteditable
contextmenu
dir
draggable
dropzone
hidden
id
itemprop
lang
spellcheck

We should consider adding almost all of them, or keying this off of element type.